content/millennium/r4/request-and-response/communication.md

```diff
@@ -0,0 +1,248 @@
+* Modifier fields should not be provided, and will cause the transaction to fail.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:communication, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Communication
+
+#### Body
+
+<%= json(:r4_communication_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/json
+Date: Tue, 04 Aug 2020 17:59:03 GMT
+Etag: W/"0"
+Location: https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Communication/472894781.0.-4.0
+Last-Modified: Tue, 04 Aug 2020 17:59:03 GMT
+Server-Response-Time: 22.810143
+Status: 201 Created
+Vary: Origin
+X-Request-Id: b89310ef-51ef-4b76-b92e-d070496c999c
+X-Runtime: 0.022706
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Patch
+
+Patch an existing communication.
+
+    PATCH /Communication/:id
+
+_Implementation Notes_
+
+* This implementation follows the [JSON PATCH](https://tools.ietf.org/html/rfc6902) spec.
+* Only operations on the paths listed below are supported.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Accept': 'application/fhir+json',
+                   'Content-Type': 'application/json-patch+json', 'If-Match': 'W/"&lt;Current version of the Communication resource>"'} %>
+
+### Patch Operations
+
+<%= patch_definition_table(:communication_patch, :r4) %>
+
+### Example
+
+#### Request
+
+    PATCH https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Communication/123
+
+#### Body
+
+<%= json(:r4_communication_patch) %>
+
+#### Response
+
+<%= headers status: 200 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/json
+Date: Wed, 11 Nov 2020 18:42:40 GMT
+Etag: W/"2"
+Last-Modified: Wed, 11 Nov 2020 18:42:40 GMT
+Server-Response-Time: 826.746885
+Status: 200 OK
+Vary: Origin
+X-Request-Id: 1594889a-8278-43f2-9c83-d84d6d2b3c72
+X-Runtime: 0.826616
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+[`reference`]: https://hl7.org/fhir/R4/search.html#reference
+[`token`]: https://hl7.org/fhir/R4/search.html#token
+[`date`]: http://hl7.org/fhir/R4/search.html#date
```

Comment:
this and codeableconcept need to be https.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/607/commits/396b90d0f3456da2649be53c3e9a900e7001e024#diff-cc6586eb639048bdea01208ce786612c64361622cd67bd53a5bf7253f27a840aL245